clinica/iotools/bids_utils.py

```diff
@@ -435,19 +435,16 @@ def write_modality_agnostic_files(study_name, bids_dir):
     from os import path
 
     import clinica
+    from clinica.iotools.bids_dataset_description import BIDSDatasetDescription
 
-    dataset_dict = {"Name": study_name, "BIDSVersion": "1.4.1", "DatasetType": "raw"}
-    dataset_json = json.dumps(dataset_dict, skipkeys=True, indent=4)
-    f = open(path.join(bids_dir, "dataset_description.json"), "w")
-    f.write(dataset_json)
-    f.close()
+    with open(path.join(bids_dir, "dataset_description.json"), "w") as f:
+        BIDSDatasetDescription(name=study_name, bids_version="1.4.1").write(to=f)
```

Comment:
Why are we using a hardcoded version here while we have a global variable for BIDS?

Reply:
To remain consistent with the old code. I guess BIDS version `1.4.1`was the standards against which the first converters were coded and no one checked whether they are compatible with the latest standards version. If we are confident they are indeed compliant with the latest version, then we could drop setting it explicitly.